recipes/napari-ndtiffs/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "napari-ndtiffs" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-ndtiffs-{{ version }}.tar.gz
+  sha256: 39a75ef7b7a2b3be4b0b017ffcd3bedffeab47363418491a4ce848316d864939
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+  run:
+    - dask-core
+    - napari-plugin-engine >=0.1.4
+    - numpy
+    - python >=3.7
+    - python-dateutil
+    - scipy
+    - tifffile
```

Comment:
Do we want to add `pyopencl` here?

Reply:
Yes please, can definitely take advantage